docs/support-ports.md

```diff
@@ -0,0 +1,14 @@
+# Supported Port for eks or ks3
+
+|  Port          |   Protocol    | Description 
+| -------------- | ------------- | -------------
+| 6443           | TCP           | Kubernetes API server. Required for communication between the Kubernetes
+|                |               | control plane and the nodes in the cluster.
```

Comment:
please combine line 5-6 to a single line. So that when you view it, shows in a single line
<img width="763" alt="Screenshot 2024-05-30 at 17 12 05" src="https://github.com/simplyblock-io/spdk-csi/assets/9148277/6cf88acf-75f8-412e-85d0-98737f5a4a8d">

Reply:
similarly at other places in the PR

Reply:
Alright